src/components/EnsureDeviceAppInteraction.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import type { Account } from '@ledgerhq/live-common/lib/types'
+import { getCryptoCurrencyIcon } from '@ledgerhq/live-common/lib/react'
+
+import { createCancelablePolling } from 'helpers/promise'
+import { standardDerivation } from 'helpers/derivations'
+import { isSegwitAccount } from 'helpers/bip32'
+import DeviceInteraction from 'components/DeviceInteraction'
+import getAddress from 'commands/getAddress'
+import IconUsb from 'icons/Usb'
+
+import type { Device } from 'types/common'
+
+import { getCurrentDevice } from 'reducers/devices'
+
+const usbIcon = <IconUsb size={36} />
+
+const mapStateToProps = state => ({
+  device: getCurrentDevice(state),
+})
+
+class EnsureDeviceAppInteraction extends Component<{
+  device: ?Device,
+  account: Account,
+}> {
+  render() {
+    const { account, ...props } = this.props
+    const Icon = getCryptoCurrencyIcon(account.currency)
+    return (
+      <DeviceInteraction
+        steps={[
+          {
+            id: 'device',
+            title: 'Connect your device',
+            icon: usbIcon,
+            desc: 'Because it is required',
+            minMs: 300,
+            run: () =>
+              createCancelablePolling(500, () => {
```

Comment:
each step `run` function should be set on the instance & not recreated each time. 
(especially fact it access `this.props` from inside this can be error prone)